From 4dbfc1040179cee4f532f44a9b82e6001589372c Mon Sep 17 00:00:00 2001
From: Daniel Baluta <daniel.baluta@nxp.com>
Date: Wed, 10 May 2017 16:46:00 +0300
Subject: [PATCH 1766/5242] MLK-14841: ARM: dts: Fix DAI CPU node name

commit  4705151d33e6ef8b5f5ffb687100b86d744663a4 from
https://source.codeaurora.org/external/imx/linux-imx.git

For wm8962 we use imx-wm8962 machine driver which expects DAI CPU node
name to be "cpu-dai".

Signed-off-by: Daniel Baluta <daniel.baluta@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sx-sdb-sai.dts |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx6sx-sdb-sai.dts b/arch/arm/boot/dts/imx6sx-sdb-sai.dts
index c1a7b325..614cffe 100644
--- a/arch/arm/boot/dts/imx6sx-sdb-sai.dts
+++ b/arch/arm/boot/dts/imx6sx-sdb-sai.dts
@@ -43,7 +43,7 @@
 
 / {
 	sound {
-		audio-cpu = <&sai1>;
+		cpu-dai = <&sai1>;
 	};
 };
 
-- 
1.7.9.5

